### DISCUSSION TITLE
A mild curiosity

### DISCUSSION THREAD

**Discussion by Prototaxis** [Anarios/return-youtube-dislike#690] [Show and tell]
![image](https://user-images.githubusercontent.com/59053343/180457305-1122a035-19ec-47d7-b464-f95f310e3668.png)

**Comment by sy-b** [comment]
@Prototaxis
Please elaborate

Where is it?

**Reply by Prototaxis** [reply]
Well obviously I can't do that in public but if the main devs are interested I could provide the needed info privately.

**Reply by Anarios** [reply]
@Prototaxis, you can reach me at selivano.d@gmail.com
